.gitignore

```diff
@@ -1,7 +1,6 @@
 # Specifies files to intentionally ignore when using Git
 # http://git-scm.com/docs/gitignore
 
-src/lib/
```

Comment:
I'm going to wait until we have our meeting Friday so we can confirm with the rest of the team.  As I said I'm not against this.